apps/nowcasting-app/components/charts/remix-line.tsx

```diff
@@ -483,8 +483,7 @@ const RemixLine: React.FC<RemixLineProps> = ({
                         typeof value[1] !== "number"
                       )
                         return null;
-                      if ((key === "PROBABILISTIC_RANGE" && value[0] < 0) || value[1] < 0)
-                        return null;
+                      if (key === "PROBABILISTIC_RANGE" && (value[0] < 0 || value[1] < 0)) return 0;
```

Comment:
hi @braddf, just a little bit of context on this line of code. the `PROBABILISTIC_RANGE` arrives in the Tooltip as an array of two values that get plotted to create the the `Area` component in the `remix-line.tsx`. will returning `0` here cause any problems?